app/utils/MCLaunchCommand.js

```diff
@@ -3,11 +3,13 @@ import fs from 'fs';
 import path from 'path';
 import { promisify } from 'util';
 import log from 'electron-log';
+import { connect } from 'react-redux';
+import javaArg from '../components/Settings/components/JavaManager/JavaArguments';
```

Comment:
Why importing a component into a utility function not directly related to react?